### PR TITLE
Upgrades to v0.10.1

### PR DESCRIPTION
**Pull request by jqyu**
https://github.com/purescript/purescript/releases/tag/v0.10.1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
     "build": "node build.js",
     "lint": "eslint --config @shinnn/node --ignore-path .gitignore .",
     "pretest": "npm run-script lint && npm run-script build",
-    "test": "node --strong_mode test.js",
```
Comment (shinnn) [inline package.json]: When you remove `--strong_mode` flag, you also need to remove `use strong` directive. https://github.com/purescript-contrib/node-purescript-bin/commit/ca0a7ba31770bd2ec552df436eb91048d6525a74#diff-1dd241c4cd3fd1dd89c570cee98b79ddL1

Comment (shinnn) [inline package.json]: Fixed in https://github.com/purescript-contrib/node-purescript-bin/commit/ca0a7ba31770bd2ec552df436eb91048d6525a74

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "purescript",
-  "version": "0.9.3",
+  "version": "0.10.1",
```
Comment (shinnn) [inline package.json]: https://twitter.com/sindresorhus/status/298376330544824320

Comment (jqyu) [inline package.json]: Whoops! Thanks for the tip.

**Comment by shinnn** [comment]
Thanks, @jqyu!

**Comment by shinnn** [comment]
I just published v0.10.1 to npm.

**Comment by garyb** [comment]
Thanks both! ❤️

**Pull request merged**